src/SoftwareSerial.cpp

```diff
@@ -228,7 +271,8 @@ void ICACHE_RAM_ATTR SoftwareSerial::preciseDelay(uint32_t deadline, bool asyn)
 	if (asyn && !m_intTxEnabled) { interrupts(); }
 	int32_t micro_s = static_cast<int32_t>(deadline - ESP.getCycleCount()) / ESP.getCpuFreqMHz();
 	if (micro_s > 0) {
-		if (asyn) optimistic_yield(micro_s); else delayMicroseconds(micro_s);
+		if (asyn) optimistic_yield(micro_s); 
+		else delayMicroseconds(micro_s);
```

Comment:
Please undo pure white-space changes, they can make merging a mess - I am still actively improving timings all over the code.

Reply:
Fixed